code/control/DeployPlanDispatcher.php

```diff
@@ -124,15 +125,18 @@ public function gitrefs(\SS_HTTPRequest $request) {
 	}
 
 	/**
-	 * Generate the data structure used by the frontend component.
-	 *
-	 * @param string $name of the component
+	 * @param string $name
 	 *
 	 * @return array
 	 */
-	public function getModel($name) {
+	public function getModel($name = '') {
 		return [
-			'APIEndpoint' => Director::absoluteBaseURL().$this->Link()
+			'namespace' => self::ACTION_PLAN,
```

Comment:
Send over information to the front end about which dispatcher this is, so we can have multiple dispatchers and now mess up the CSRF tokens